systemvm/scripts/config_ssl.sh

```diff
@@ -204,6 +203,7 @@ if [ -f "$customCACert" ]
 then
   keytool -delete -alias $aliasName -keystore $keyStore -storepass $storepass -noprompt
   keytool -import -alias $aliasName -keystore $keyStore -storepass $storepass -noprompt -file $customCACert
+  keytool -importkeystore -srckeystore $defaultJavaKeyStoreFile -destkeystore $keyStore -srcstorepass $defaultJavaKeyStorePass -deststorepass $storepass
```

Comment:
It seems that we absolutely need a -noprompt option here, since re-applying the configuration will lead to keytool prompting whether to overwrite or not, and freezing the Agent then.